### ISSUE TITLE
Gravis ultrasound -> Duke Nukem -> Stereo sound on right speaker only

### ISSUE THREAD

**Issue by Choum28** [joncampbell123/dosbox-x#1029]
**Describe the bug**
On duke nukem 3d (v1.5 atomic edition), with the gravis ultrasound emulation on, the Digital sound is playing only on the right speakers instead of stereo.
The problem do not appears during the Setup of the sound card, the autotest and stereo test is ok.

**Expected behavior**
Stereo sound ingame.

**Environment (please complete the following information):**
tested on dosbox-x-windows-20190301-184734
Win32 version SDL1
Gravis ultrasound software 4.11
Pro Patches Lite v1.61

**Comment by joncampbell123** [comment]
If you run setup first then the game, do the sound effects play centered?

**Comment by Choum28** [comment]
No the sound FX still play on the right speakers only.
I also try to reduce the game setting to one voice and r8khz, no change.

gus=true
autoamp=false
unmask dma=false
pic unmask irq=false
startup initialized=false
dma enable on dma control polling=false
clear dma tc irq if excess polling=false
force master irq enable=false
gus panning table=default
gusrate=44100
gus fixed render rate=false
gusmemsize=-1
gus master volume=0.00
gusbase=240
gusirq=5
gusdma=3
irq hack=none
gustype=classic
ultradir=C:\ULTRASND

**Comment by Allofich** [comment]
> On duke nukem 3d (v1.5 atomic edition)

I don't have that installed, but I do have the Duke Nukem 3d shareware (version 1.3D), and the sounds play in both channels for me with Gravis Ultrasound, using the same settings in dosbox.conf as you posted.

In sound setup I picked all default options for the Gravis Ultrasound (8 voices, 16 bit mixing, stereo sound, 22 Khz)

For MIDI I selected General Midi with default MIDI port 0x330.

I used a build from February, then the latest code as of now. Both worked.

Do you have the problem in the shareware version? If you do, that is probably easier to acquire and test with (maybe a moot point if joncampbell123 already has Duke Nukem 3D v1.5 atomic edition installed).

**Comment by Allofich** [comment]
> Gravis ultrasound software 4.11
> Pro Patches Lite v1.61

I do not have these, I was just using Dosbox-X with "gus=true". Also I used the Windows 64-bit version of Dosbox-X, not 32-bit version.

**Comment by Allofich** [comment]
By the way, Duke Nukem 3D has a "Flip Stereo" option in the Sounds menu. What happens if you select it? (Plays in left speaker only? Becomes silent?) It might help to know that.

**Comment by Choum28** [comment]
Flip stereo on or off do nothing SFX sound will come from left side only.
MIDI music is playing fine on both speakers.
Same problem with the shareware (1.3d) version, SFX sound left only.

> Gravis ultrasound software 4.11
> Pro Patches Lite v1.61
> 
> I do not have these, I was just using Dosbox-X with "gus=true". Also I used the Windows 64-bit version of Dosbox-X, not 32-bit version.

So your Gravis ultrasound is not installed without the Gravis ultrasound software 4.11.
Pro patches lite v1.61 is just for better midi music

installation procedure 
https://www.vogons.org/viewtopic.php?t=16974

**Comment by Allofich** [comment]
Thank you, I didn't know the Gravis Ultrasound software is required. I followed the installation procedure you linked to.

1) I can confirm that with Gravis ultrasound software 4.11 + Pro patches lite v1.61 the digital sound only comes from the right speaker. (I think you accidentally said "left" in your last reply)
2) If you do use Gravis ultrasound software 4.11, but **without**  Pro patches lite v1.61, **or** if you select an option other than "Ultrasound" in Duke Nukem 3D's setup program, the digital sound plays in both speakers.

In other words, the problem happens if you play with Pro patches lite v1.61 and "Ultrasound" selected for music in the Duke Nukem 3D setup program.

**Comment by Choum28** [comment]
yes it was on right speaker.
I confirm your finding the problems seems specific to the use of the Pro patches Lite and this game.

**Comment by Allofich** [comment]
Just tested with the latest commit, but the issue still happens.

**Comment by Allofich** [comment]
By the way, this same problem happens in DOSBox SVN (tested commit r4252), so this issue seems to have been inherited from mainline.
